src/ejabberd_c2s.erl

```diff
@@ -204,6 +204,10 @@ init([{SockMod, Socket}, Opts]) ->
     TLSOpts = verify_opts(Verify) ++ TLSOpts1,
     [ssl_crl_cache:insert({file, CRL}) || CRL <- proplists:get_value(crlfiles, Opts, [])],
     IP = peerip(SockMod, Socket),
+    StreamMgmtConfig = case gen_mod:is_loaded(?MYNAME, mod_stream_management) of
```

Comment:
This should be done when we actually know the proper server. We can afford to wait. :)